src/views/landing.njk

```diff
@@ -32,7 +32,8 @@
             isStartButton: true,
             attributes: {
               id: "submit"
-            }
+            },
+            href: penaltyDetailsPage
```

Comment:
I think we were avoiding adding transitions to other pages through the template files. Currently in master, landing page redirects through a post request (when button is clicked) to a entry controller ('/start') which calls a get request that redirects to penalty details page. This is because landing page is ultimately a throwaway page. Is this explanation correct @geek-soft ? haha

Reply:
Redirect is done on submission, you are right @akaJSaunders it is not needed.